playwright/e2e/share-dialog/share-dialog.spec.ts

```diff
@@ -35,7 +35,7 @@ test.describe("Share dialog", () => {
 
         const rightPanel = await app.toggleRoomInfoPanel();
         await rightPanel.getByRole("menuitem", { name: "People" }).click();
-        await rightPanel.getByRole("button", { name: `${user.userId} (power 100)` }).click();
+        await rightPanel.getByRole("gridcell", { name: user.displayName }).click();
```

Comment:
Not sure this is great in terms of a11y semantics, a 1-col grid is definitely not the most appropriate role set to use